README.md

```diff
@@ -1 +1,62 @@
-# Todo_review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do list
+
+> This project is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## Demo 
+Click here to see [Live](https://lionrouge1.github.io/To_do_list/)
```

Comment:
Removing this live link can help but you can also put the live link

Reply:
👍

Reply:
Do you mind adding the link to your live page here